mifosng-android/src/main/java/com/mifos/mifosxdroid/adapters/IndividualCollectionSheetAdapter.java

```diff
@@ -0,0 +1,282 @@
+package com.mifos.mifosxdroid.adapters;
+
+import android.content.Context;
+import android.support.v7.widget.RecyclerView;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.Spinner;
+import android.widget.TableLayout;
+import android.widget.TextView;
+
+import com.mifos.api.model.BulkRepaymentTransactions;
+import com.mifos.mifosxdroid.R;
+import com.mifos.mifosxdroid.online.collectionsheetindividual.OnRetrieveSheetItemData;
+import com.mifos.objects.accounts.loan.PaymentTypeOptions;
+import com.mifos.objects.collectionsheet.LoanAndClientName;
+import com.mifos.objects.collectionsheet.LoanCollectionSheet;
+
+import java.util.List;
+import java.util.Locale;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+/**
+ * Created by Tarun on 17-07-2017.
+ */
+
+public class IndividualCollectionSheetAdapter extends
+        RecyclerView.Adapter<IndividualCollectionSheetAdapter.ViewHolder> {
+
+    private List<String> paymentTypeList;
+    private List<LoanAndClientName> loans;
+    private List<PaymentTypeOptions> paymentTypeOptionsList;
+    private Context c;
+
+    private OnRetrieveSheetItemData sheetItemClickListener;
+
+    public IndividualCollectionSheetAdapter() {
```

Comment:
Inject this class like this https://github.com/openMF/self-service-app/blob/development/app/src/main/java/org/mifos/selfserviceapp/ui/adapters/CheckBoxAdapter.java and you can use Application Context too.